tests/test_pybats.py

```diff
@@ -316,18 +497,27 @@ class TestBats2d(unittest.TestCase):
                  'wy':0.0, 'u':1285.6114501953125}
     knownMax2 = {'jx': 1.680669083725661e-05, 'jbz': 8.276679608343329e-08,
                  'wy': 0.0, 'u': 1285.6114501953125}
+    knownCalcSingle = [8.6176513671875, 9.724967e-11, 1.0006003, 400.0,
+                       0.0067278803, 0.009994, 0.99992746, 399.99097, -2.691152,
+                       0.4002311, 8.664787, 9.730319e-17, 9.760558]
+    calcnames = ['t', 'j', 'b', 'u', 'bx_hat', 'by_hat', 'bz_hat',
+                 'u_perp', 'u_par', 'E', 'beta', 'jb', 'alfven']
 
     def setUp(self):
-        self.mhd = pbs.Bats2d(os.path.join(spacepy_testing.datadir, 'pybats_test',
-                                           'y=0_mhd_1_e20140410-000050.out'))
-        self.outs = pbs.Bats2d(os.path.join(spacepy_testing.datadir, 'pybats_test',
+        self.mhd = pbs.Bats2d(os.path.join(spacepy_testing.datadir,
+                                           'pybats_test', 'y0_ascii.out'))
+        self.outs = pbs.Bats2d(os.path.join(spacepy_testing.datadir,
+                                            'pybats_test',
                                             'y=0_mhd_1_e20140410-000000-000_' +
                                             '20140410-000200-000.outs'))
 
-    def testCalc(self):
+    def testCalcSingle(self):
         # Test all calculations:
         self.mhd.calc_all()
 
+        for v, x in zip(self.calcnames, self.knownCalcSingle):
+            self.assertAlmostEqual(x, self.mhd[v][0], places=4)
+
```

Comment:
knownCalcSingle was added in the original at spacecataz/spacepy@9cc1c1a66a6dc8be749c960e017a025a7b7b86ac "Fixed tests and addressed bugs", which is equivalent to 0f5c628ca9dffeaee061ded96a483662aaf5fdd0 in the rebase.

Reply:
Because the addition of knownCalcSingle and testCalcSingle were sort of smeared across commits in both the original and the rebase, we're not going to worry about matching the smearing between the original and rebase, it's not worth cleaning up.